src/lib/dates.ts

```diff
@@ -36,10 +42,47 @@ export const DEFAULT_DURATIONS: DurationOption[] = [
   { label: 'Days ago', value: 'day' },
 ];
 
+export const CUSTOM_DATE_RANGE_LABEL_SEPARATOR =
+  '<i aria-hidden="true" class="fa fa-arrow-right"></i>';
```

Comment:
Maybe using the unicode character → is enough, and removes the need to return HTML there.